src/MySql/custom/Test-AzMySqlFlexibleServerConnect.ps1

```diff
@@ -0,0 +1,164 @@
+        ${InputObject},
+
+        [Parameter(HelpMessage = 'Administrator username for the server. Once set, it cannot be changed.')]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Body')]
+        [System.String]
+        ${AdministratorUserName},
+
+        [Parameter(Mandatory, HelpMessage = 'The password of the administrator. Minimum 8 characters and maximum 128 characters. Password must contain characters from three of the following categories: English uppercase letters, English lowercase letters, numbers, and non-alphanumeric characters.')]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Body')]
+        [System.Security.SecureString]
+        [ValidateNotNullOrEmpty()]
+        ${AdministratorLoginPassword},
+
+        [Parameter(HelpMessage = 'The credentials, account, tenant, and subscription used for communication with Azure.')]
+        [Alias('AzureRMContext', 'AzureCredential')]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Azure')]
+        [System.Management.Automation.PSObject]
+        ${DefaultProfile},
+
+        [Parameter(DontShow, HelpMessage = 'Wait for .NET debugger to attach.')]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Runtime')]
+        [System.Management.Automation.SwitchParameter]
+        ${Break},
+
+        [Parameter(DontShow)]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Runtime')]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Runtime.SendAsyncStep[]]
+        # SendAsync Pipeline Steps to be appended to the front of the pipeline.
+        ${HttpPipelineAppend},
+
+        [Parameter(DontShow)]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Runtime')]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Runtime.SendAsyncStep[]]
+        # SendAsync Pipeline Steps to be prepended to the front of the pipeline.
+        ${HttpPipelinePrepend},
+
+        [Parameter(DontShow)]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Runtime')]
+        [System.Uri]
+        # The URI for the proxy server to use.
+        ${Proxy},
+
+        [Parameter(DontShow)]
+        [ValidateNotNull()]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Runtime')]
+        [System.Management.Automation.PSCredential]
+        # Credentials for a proxy server to use for the remote call.
+        ${ProxyCredential},
+
+        [Parameter(DontShow)]
+        [Microsoft.Azure.PowerShell.Cmdlets.MySql.Category('Runtime')]
+        [System.Management.Automation.SwitchParameter]
+        # Use the default credentials for the proxy.
+        ${ProxyUseDefaultCredentials}
+    )
+
+    process {
+        if (!(Get-Module -ListAvailable -Name SimplySQL)){
+            Write-Error "This cmdlet requires SimplySQL module. Please install the module first by running Install-Module -Name SimplySQL."
```

Comment:
Neet to exit here.

Reply:
@DaeunYim could you modify it in the generation branch?

Reply:
Made the change for both mysql and pg